### DISCUSSION TITLE
How to get back modifiers on consumables etc?

### DISCUSSION THREAD

**Discussion by TheCToTheY** [Pryaxis/TShock#2810] [Q&A] [answered]
Hello, in my Adventure/RP map, I used to do things such as give grenades the broken modifier, or some arrows the damaged modifier so some "high end" items could still be used in the challenges I put forward and not be broken, presenting more variable gameplay and adding variety in general. Also, armor with modifiers.

I assume this was made not possible anymore because of potential cheating issues, but for me and my small group of friends in my turn-based RP world, this was a godsend. Is there any way I can somehow enable this on my end with some setting or the other?

Thanks.

**Comment by TheCToTheY** [comment]
Also, whenever I try to do /wind in game, I get this error and my game crashes. Though the same command (/wind 50) works from the server terminal. I do wind command with rain to spawn a thunderstorm, since I don't think there is a /worldevent thunderstorm command. Could we get one?

https://prnt.sc/aMLNdfreDsQi

**Comment by drunderscore** [comment, marked as answer]
The ability to place modifiers on items that cannot obtain said modifier was removed by Re-Logic -- we cannot re-implement it.

**Reply by punchready** [reply]
"We made this change precisely because of people exploiting it in MP." - An anonymous ReLogic employee after being asked to revert this change for the sake of multiplayer. - We aren't gonna get it back.

**Reply by TheCToTheY** [reply]
I see, horrible news then :(
